packages/api/tests/endpoints/collections/update-collection.js

```diff
@@ -117,7 +154,7 @@ test('CUMULUS-912 PUT with pathParameters and with an invalid access token retur
 
 test.todo('CUMULUS-912 PUT with pathParameters and with an unauthorized user returns an unauthorized response');
 
-test('PUT replaces an existing collection', async (t) => {
+test.serial('PUT replaces an existing collection and sends an SNS message', async (t) => {
```

Comment:
Do these tests need to be `serial`? If the SNS topic and queue that are used per test have randomized names, then there shouldn't be test collisions

Reply:
I was very wrong